lib/ambi.ex

```diff
@@ -14,12 +14,13 @@ defmodule Ambi do
   alias Phoenix.PubSub
 
   def add_reading(attrs \\ %{}) do
-    %Ambi.Reading{}
+    result = %Ambi.Reading{}
```

Comment:
I forget, does Elixir return the last line by default? If so, you could avoid thid assign and just move the logic below the log line